unstructured_ingest/v2/processes/connectors/lancedb/aws.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+
+from pydantic import Field, Secret
+
+from unstructured_ingest.v2.interfaces.connector import AccessConfig
+from unstructured_ingest.v2.processes.connector_registry import DestinationRegistryEntry
+from unstructured_ingest.v2.processes.connectors.lancedb.lancedb import (
+    LanceDBRemoteConnectionConfig,
+    LanceDBUploader,
+    LanceDBUploaderConfig,
+    LanceDBUploadStager,
+    LanceDBUploadStagerConfig,
+)
+
+CONNECTOR_TYPE = "lancedb_s3"
```

Comment:
Just to match how we'e been adding the cloud provider to the connector names elsewhere, such as databricks volumes. 
```suggestion
CONNECTOR_TYPE = "lancedb_aws"
```